src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -907,6 +917,17 @@ def aks_update(cmd,     # pylint: disable=too-many-statements,too-many-branches,
                enable_apiserver_vnet_integration=False,
                apiserver_subnet_id=None,
                ):
+
+    if disable_disk_driver or disable_file_driver:
+        msg = "Please make sure there are no existing PVs and PVCs that are used by the CSI driver before disabling."
+        if not yes and not prompt_y_n(msg, default="n"):
+            return None
+    if disable_snapshot_controller:
+        msg = "Please make sure there are no existing VolumeSnapshots, VolumeSnapshotClasses and VolumeSnapshotContents CRDs " \
```

Comment:
should remove `CRDs` since those objects are created by AKS:
```
"Please make sure there are no existing VolumeSnapshots, VolumeSnapshotClasses and VolumeSnapshotContents 
 that are used by the snapshot controller before disabling."
```

Reply:
Done